src/main/fruits.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
```

Comment:
Checklist: Place the input and output files into the src/main/resources folder.

Reply:
fixed